src/components/Auth/UserFormDialog/controls/CustomAttributes.tsx

```diff
@@ -81,6 +81,7 @@ export const CustomAttributes: React.FC<
           target="_blank"
           rel="noopener noreferrer"
           href="https://firebase.google.com/docs/auth/admin/custom-claims"
+          aria-label="Learn more about custom key value attributes."
```

Comment:
nit: Consider using `title` here since the text is helpful for most user agents (such as mouse hover tooltip), not just screen readers